pkg/analysis/helpers/extractjsontags/analyzer.go

```diff
@@ -81,6 +81,7 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return results, nil
 }
 
+//nolint:cyclop
 func extractTagInfo(tag *ast.BasicLit) FieldTagInfo {
```

Comment:
```
pkg/analysis/helpers/extractjsontags/analyzer.go:84:1: calculated cyclomatic complexity for function extractTagInfo is 12, max is 10 (cyclop)
func extractTagInfo(tag *ast.BasicLit) FieldTagInfo {
```

I've noticed a linter error for `extractTagInfo`. Can I add nolint  for this func?